Source/Initialization/WarpXInitData.cpp

```diff
@@ -573,6 +574,12 @@ WarpX::InitData ()
         // Loop through species and calculate their space-charge field
         bool const reset_fields = false; // Do not erase previous user-specified values on the grid
         ExecutePythonCallback("beforeInitEsolve");
+
+        // Run div cleaner here on the loaded fields
+        if (WarpX::do_divb_cleaning_external) {
+            WarpX::ProjectionCleanDivB();
+        }
```

Comment:
After we merge #5162, these lines could be moved to `WarpX::LoadExternalFields`.

Reply:
Sounds good. I will wait for that to be merged.

Reply:
I moved this out of the check for checkpoint restart so it will run for new starts and restarts, however there were uninitialized values when trying to run from the LoadExternalFields routine, so I was unable to move to that location. I think having it run and cleaning it up prior to running initial solutions is a good spot.